tests/psp1155/extensions/burnable.tests.ts

```diff
@@ -39,38 +38,16 @@ describe('MY_PSP1155_BURNABLE', () => {
         await expect(query.balanceOfBatch([[alice.address, token1], [alice.address, token2]]))
             .to.have.output([0, amount2])
 
-        await contract.tx.burn([[token1, amount1], [token2, amount2]])
+        await contract.tx.burn(sender.address, [[token1, amount1], [token2, amount2]])
         await fromSigner(contract, alice.address).tx.setApprovalForAll(sender.address, true)
```

Comment:
You don't need setApprovalForAll as you removed this check in `burn`. Please update it in **ALL** cases.